near-sdk-macros/src/lib.rs

```diff
@@ -44,7 +49,15 @@ use syn::{parse_quote, File, ItemEnum, ItemImpl, ItemStruct, ItemTrait, WhereCla
 /// }
 /// ```
 #[proc_macro_attribute]
-pub fn near_bindgen(_attr: TokenStream, item: TokenStream) -> TokenStream {
+pub fn near_bindgen(attrs: TokenStream, item: TokenStream) -> TokenStream {
+    let attr_args = parse_macro_input!(attrs as AttributeArgs);
+    let config = match MacroConfig::from_list(&attr_args) {
+        Ok(args) => args,
+        Err(e) => {
+            return TokenStream::from(e.write_errors());
+        }
+    };
```

Comment:
To extend on the motivation for #951: Previously attributes passed to `near_bindgen` were ignored. Now `darling` tries to parse them into `MacroConfig`. The tokens `init => new` in the compilation test were previously ignored. Now this test will fail since `init => new` cannot be parsed into `MacroConfig`.